### PR TITLE
Updates readme with 2019 dates

### LINKED ISSUE #9: Pick dates
We are winnowing down dates based on organizer availability here: https://hackmd.io/NAHVnJD4So6YYv8Jnn0geg?edit#

### PR DESCRIPTION
**Pull request by dcwalk**
This closes #9, might want to sit on the LOA signing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # State of Our Networks 2019
 
-This repository has organizing materials for the upcoming State of Our Networks, for any questions email orga@ournetworks.ca!
+This repository has organizing materials for the Our Networks **RE: Infrastructures** on **September 20-22, 2019**, for any questions email orga@ournetworks.ca!
+
+**September 19** - Load-in // Workshopping
+**September 20-22** - Conference, [Toronto Media Arts Centre](https://www.tomediaarts.org/), 32 Lisgar Street  
+**September 20-24** - Exhibition
+
+Our hashtags are #REInfrastructures, #OurNetworks, or #OurNetworks2019 (i.e., on [Twitter](https://twitter.com/search?q=%23OurNetworks&src=typd)) and we love the :control_knobs: `:control_knobs:` , :satellite: `:satellite: or :satellite_antenna:` , :loudspeaker: `:loudspeaker:` and :artificial_satellite: `:artificial_satellite:` emojis.
```
Comment (benhylau) [inline README.md]: `#REInfra`?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # State of Our Networks 2019
 
-This repository has organizing materials for the upcoming State of Our Networks, for any questions email orga@ournetworks.ca!
+This repository has organizing materials for the Our Networks **RE: Infrastructures** on **September 20-22, 2019**, for any questions email orga@ournetworks.ca!
+
+**September 19** - Load-in // Workshopping
+**September 20-22** - Conference, [Toronto Media Arts Centre](https://www.tomediaarts.org/), 32 Lisgar Street  
+**September 20-24** - Exhibition
```
Comment (benhylau) [inline README.md]: Is it clear that Exhibition is same venue?

Comment (dcwalk) [reply README.md]: I think it is fine...? Also we're not representing it like this on the website, which by and large is where folks look

File: README.md (hunk 1 above)
Comment (dcwalk) [reply README.md]:
```suggestion
Our hashtags are #REInfra, #REInfrastructures, #OurNetworks, or #OurNetworks2019 (i.e., on [Twitter](https://twitter.com/search?q=%23OurNetworks&src=typd)) and we love the :control_knobs: `:control_knobs:` , :satellite: `:satellite: or :satellite_antenna:` , :loudspeaker: `:loudspeaker:` and :artificial_satellite: `:artificial_satellite:` emojis.
```

**Comment by dcwalk** [comment]
Merging this in!

**Pull request merged**